framework/src/media/audio/audio_manager.c

```diff
@@ -1349,22 +1652,25 @@ audio_manager_result_t change_stream_device(int card_id, int device_id, audio_io
 
 	get_card_path(path, card_id, device_id, direct);
 
-	/* check path is available as a driver path */
+	/* check target path is available as a driver path */
```

Comment:
target -> the target ..?

Reply:
'process' is abstract noun, in this case the is not needed.